src/domain/app/header/navbar/Navbar.tsx

```diff
@@ -15,11 +15,15 @@ const Navbar: React.FunctionComponent = props => {
     <div className={styles.navbarTop}>
       <div className={styles.logoWrapper}>
         <div className={styles.logo} onClick={() => history.push('/')}></div>
-        <h3 className={styles.appName}>{t('appName')}</h3>
+        {!isSmallScreen ? (
```

Comment:
!IsSmallScreen && ...
No need return ''

Reply:
Good tip, thank you.